client/src/routes/Home/images/r.js

```diff
@@ -0,0 +1,324 @@
+for (let j in rows)
+{
+  L.marker(rows[j][0],rows[j][1]).addTo(map);
+  console.log(rows[j][0]);
+};
+  }, [])
+
+  return (
+    <div className="App">
+    <h1 style={{textAlign: "center"}}>Aqi Map</h1>
+      <Map  ref={mapRef} center={[28.7041, 77.1025]} zoom={10} style={{height:"85vh"}}>
+        <TileLayer url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png" attribution="&copy; <a href=&quot;https://www.openstreetmap.org/copyright&quot;>OpenStreetMap</a> contributors" />
+        <Search/>
+      </Map>
+    </div>
+  );
+}
+
+export default App;
+
+
+
+
+
+
+
+// real code
+
+
+function App() {
+  const mapRef = useRef();
+  let rows = [], columns;
+  let aqiGradient = {
+    0: "#F9F9F9",
+    10: "#CCCCFF",
+    30: "#BBBBEE",
+    60: "#AAAADD",
+    80: "#9999CC",
+    100: "#8888BB",
+
+    120: "#90FA96",
+    140: "#82EA64",
+    160: "#66DA36",
+    180: "#50CA2C",
+    200: "#4ABA26",
+
+    220: "#FAFA5D",
+    240: "#EAEA46",
+    260: "#DADA4D",
+    280: "#CACA42",
+    300: "#BABA36",
+
+    320: "#FF7777",
+    340: "#EE6666",
+    360: "#DD5555",
+    380: "#CC4444",
+    400: "#BB3333",
+
+    420: "#E046E0",
+    440: "#D03DD0",
+    460: "#C032C0",
+    480: "#B026B0",
+    500: "#A01DA0"
+  };
+  //console.log(obj[0].Lat);
+
+  //document.write(rows[0][2]);
+  //const df = new DataFrame(rows, columns = ['lat', 'lon', 'aqi', 'name']);
+ 
+const fetchdata=async()=> {
+  const response=await fetch(url);
+  const data= await response.json();
+  userData = data.data
+  //console.log(userData);
+}
+
+
+
+  useEffect(() => {
+    const { current = {} } = mapRef;
+    const { leafletElement: map } = current;
+   // if(!query) return;
+   fetchdata().then(() => {
+    console.log(userData);
+    for (let i in userData) {
+      if(!isNaN(userData[i].aqi)) {
+        rows.push([userData[i].lat, userData[i].lon, parseInt(userData[i].aqi)]);
+      }
+    }
+
+    if (!map) return;
+    console.log(rows);
+
+    var idw = L.idwLayer(rows, {
+      opacity: 0.4,
+     // maxZoom: 18,
+      cellSize: 1,
+      exp: 5,
+      max: 300,
+     // gradient: aqiGradient,
+    }).addTo(map);
+    // idw.addTo(map);
+    for (let j in rows){
+        // latLng.lat=rows[j][0];
+         ///latLng.lng=rows[j][1];
+         L.circleMarker([parseFloat(rows[j][0]),parseFloat(rows[j][1])]).addTo(map);
+       
+       //console.log(rows[j][0]);
+     };
+    // L.marker(rows[0][0],rows[0][1]).addTo(map);
+
+   })
+   .finally( () => {
+     setInterval(fetchdata,10000);
+   })
+
+    
+  }, [])
+
+  console.log(rows);
+
+  
+  return (
+    <div className="App">
+      <Map ref={mapRef} center={[28.7041, 77.1025]} zoom={9} >
+        <TileLayer url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png" attribution="&copy; <a href=&quot;https://www.openstreetmap.org/copyright&quot;>OpenStreetMap</a> contributors" />
+        <Search />
+        
+      </Map>
+      <div id="navigation"><div id="icon1"><img src={"/Vector.png"}></img>
```

Comment:
Move all these images to a folder in `client/assets/images/`.
Refer to this documentation for using images. https://create-react-app.dev/docs/adding-images-fonts-and-files/
use import statements.